With c++14 compiling anything against these headers gets the following error:

npm ERR! /usr/include/electron/v8-maybe.h:106:45: error: 'is_lvalue_reference_v' is not a member of 'std'; did you mean 'is_lvalue_reference'?
npm ERR!   106 |   template <class U, std::enable_if_t<!std::is_lvalue_reference_v<U>>*>
npm ERR!       |                                             ^~~~~~~~~~~~~~~~~~~~~


--- a/third_party/electron_node/common.gypi	2022-09-16 12:49:53.208366401 +0200
+++ b/third_party/electron_node/common.gypi	2022-09-19 12:14:32.359490504 +0200
@@ -440,7 +440,7 @@
       }],
       [ 'OS in "linux freebsd openbsd solaris android aix cloudabi"', {
         'cflags': [ '-Wall', '-Wextra', '-Wno-unused-parameter', ],
-        'cflags_cc': [ '-fno-rtti', '-fno-exceptions', '-std=gnu++14' ],
+        'cflags_cc': [ '-fno-rtti', '-fno-exceptions', '-std=gnu++17' ],
         'defines': [ '__STDC_FORMAT_MACROS' ],
         'ldflags': [ '-rdynamic' ],
         'target_conditions': [
@@ -580,7 +580,7 @@
           ['clang==1', {
             'xcode_settings': {
               'GCC_VERSION': 'com.apple.compilers.llvm.clang.1_0',
-              'CLANG_CXX_LANGUAGE_STANDARD': 'gnu++14',  # -std=gnu++14
+              'CLANG_CXX_LANGUAGE_STANDARD': 'gnu++17',  # -std=gnu++17
               'CLANG_CXX_LIBRARY': 'libc++',
             },
           }],
@@ -653,7 +653,7 @@
           '-qASM',
         ],
         'cflags_cc': [
-          '-qxclang=-std=c++14',
+          '-qxclang=-std=c++17',
         ],
         'ldflags': [
           '-q64',
